2016/opencv_tweakapp/main.cpp

```diff
@@ -75,7 +75,7 @@ int main()
     Mat img;
     while(true)
     {
-        int ixCurrentImage = args[5];
+        int ixCurrentImage = args[8];
```

Comment:
Change this [8] to [NUM_ARGS-1], then we won't have to keep updating it.  The last trackbar will always be the image-selector.